components/authn-service/tokens/pg/adapter.go

```diff
@@ -152,10 +152,17 @@ func (a *adapter) GetToken(ctx context.Context, id string) (*tokens.Token, error
 }
 
 func (a *adapter) GetTokenIDWithValue(ctx context.Context, value string) (string, error) {
```

Comment:
@msorens this is only used internally-- I was thinking since it's not exposed and we use it to authenticate and to migrate legacy tokens, project filtering wouldn't apply for now. What do you think?

Reply:
Discussed via the zooms: Gonna reset this